x-pack/plugins/security_solution/public/common/components/events_viewer/events_viewer.tsx

```diff
@@ -67,6 +68,8 @@ interface Props {
   sort: Sort;
   toggleColumn: (column: ColumnHeaderOptions) => void;
   utilityBar?: (refetch: inputsModel.Refetch, totalCount: number) => React.ReactNode;
+  // If truthy, the graph viewer (Resolver) is showing
```

Comment:
:grey_question: Should this come from a selector like `shouldShowResolver`?

Reply:
I only ask because I've done sparse business logic like this before and have been asked to move it to a selector to make it more readable.

Reply:
I see later maybe there _is_ a selector? It's not clear from reading this the way the comment is written.

Reply:
i'm changing some existing code here, but my thoughts:

* There is a selector (that this uses) that gets the `TimelineModel`. 
* `TimelineModel` has `graphEventId` as a field
* If `graphEventId` is falsy, there is no 'graphEvent' (aka resolver)
* If `graphEventId` ts truthy, the value is the `_id` of the document that Resolver should use.

The existing code doesn't have a specific selector for `graphEventId`. Instead it exposes the `TimelineModel` interface throughout the view.  If there was a desire to encapsulate the logic of "given a `TimelineModel`, how do I know if the graph view should show" then I would add a method to the `TimelineModel`.  I don't think that would fit the code style of the SIEM team, but I'm open to changing it if they want that.